TRANSLATION.md

```diff
@@ -1,28 +1,21 @@
 
-# Translations - internationalization (i18n)
+# Localization - internationalization (l10n/i18n)
 
 ## How to add a new language
 
-1. In `config.toml`, copy the commented section, replace XX in [[languages.XX.*]] with your language code and translate the strings.
+### First pull-request
+
+1. In `config.toml`, copy the English section (`[languages.en]`,`[languages.en.*]`), and replace `en` with your language code and translate the strings.
 2. In `i18n\` copy `en.toml` into `XX.toml` (where XX is your language code) and translate the strings.
 3. In `layouts\_default\single.html` copy the `{{ if eq .Language.Lang "fr" }}...{{ end }}` and replace `"fr"` by `"XX"` (where XX is your language code) and "Voir en Français" by the translation of "See in [your language]"
-4. In `content\XX\` (where XX is your language code), you can add your translations of `content\en\`.
-5. `content\XX\about.md` (`about/`), `content\XX\howitworks.md` (`getting-started/`), `content\XX\donate.html` (`donate/`) and `content\XX\sponsors.html` (`sponsors/`) must be translated and included in the first pull-request.
-6. Docs index (`/content/en/docs/_index.md`) should be included, at least to point users to the English documentation.
-7. After the first pull-request, more files from `content\XX\` may be translated.
-
-* Not-yet-translated pages **must not** be copied (exept `_index.md`).
-* The pull-request of a translated file must update links on other files pointing to it.
-* The *subscriber agreement* should **not** be translated.
-* The *privacy policy* (`privacy.md`) and the *Non-Discrimination Statement and Policy* (`nondiscrimination.md`) may be translated, but with a note on top saying that the English version is canonical.
-* `jobs.md` should either not be present or just contain a link to the english version.
-* Paths should stay in English: `/about` becomes `/XX/about` in every language.
-* When a file is not translated, it will not appear in the menu.
-* `cert-compat.md` and `repository.md` are not ready to be translated.
-* Blog posts (`content\XX\post\`) are not ready to be translated.
-* Images are not ready to be translated ([#314](https://github.com/letsencrypt/website/issues/314)).
-* Graphs content is not ready to be translated ([#344](https://github.com/letsencrypt/website/issues/344)).
-* The "aliases:" field in the Hugo header should not be copied. This leads to
-  inconsistent redirects.
-* `date` and `lastmod` fields in the Hugo header must correspond to the English version (and not when the translation occured)
-* You can check the translation status using https://letsencrypt.org/i18n/
+4. Copie `content\base-l10n\` into `content\XX\` (where XX is your language code)
+
+### After the first pull-request
+
+You can start translating the files that are inside `content\XX\`:
+
+* copy the English version of the file you wish to translate into `content\XX\`
+* if the header of the file contains `aliases`, **you must remove it**
```

Comment:
Be more clear about what "it" is here, what should be removed. I think you're saying they should just remove the "aliases" line, someone might misinterpret what you're saying and remove the file.

Reply:
Is the new version better?